demisto_sdk/scripts/changelog/changelog.py

```diff
@@ -69,9 +69,12 @@ def init(self) -> None:
         log = INITIAL_LOG
         log["pr_number"] = int(self.pr_number)
 
-        with (CHANGELOG_FOLDER / f"{self.pr_number}.yml").open("w") as f:
+        file_path = CHANGELOG_FOLDER / f"{self.pr_number}.yml"
+
+        with file_path.open("w") as f:
             yaml.dump(log, f)
 
+        Repo(git_path()).git.add(file_path)
```

Comment:
@MichaelYochpaz its a nice idea, but it would confuse people IMO, they won't understand how their changelog was staged.